test/test_guardrail_globals.py

```diff
@@ -168,7 +158,8 @@ def test_set_all(self):
         self.assertEqual(global_obj.mutation, True)
         self.assertEqual(global_obj.deadcode, True)
         self.assertEqual(global_obj.cycloc, True)
-        self.assertEqual(global_obj.lint_ignore, r"C:\Projects\PythonRepo\python_sample\FunctionDefExtractor\test")
+        self.assertEqual(global_obj.lint_ignore, r"C:\Projects\PythonRepo\python_sample\FunctionDefExtractor"
```

Comment:
Could you please make sure that the OS paths do not become a hindrance. Here, and in other places too?

Reply:
removed the os specific check with simple string as the test does not look for the path as they are mocked / they are checked only for reading the string arg.